src/features/header/Header.scss

```diff
@@ -23,7 +23,8 @@
   }
 
   .logo {
-    max-height: 24px;
+    max-height: 32px;
+    height: 32px;
```

Comment:
if you are gonna set the height you also need to manage the width even `width: auto` or `100%`is enough (depends on the design and space)